docs/flow/modin/experimental/reshuffling_groupby.rst

```diff
@@ -1,74 +1,8 @@
-Reshuffling GroupBy
-"""""""""""""""""""
+:orphan:
```

Comment:
no, that's why there's this `:orphan:` tag, meaning that there are no references to this page

I left it just in case someone holds a direct link to the old page, so he'll be automatically redirected to a new one instead of getting 404

Reply:
Makes sense. We will need to not forget to remove it when we completely get rid of the deprecated configs.